locust/user/task.py

```diff
@@ -384,7 +384,7 @@ def interrupt(self, reschedule=True):
         and execute, a new task.
         
         This method should not be called by the root TaskSet (the one that is immediately, 
-        attached to the User class's *task_set* attribute), but rather in nested TaskSet
+        attached to the User class's *tasks* attribute), but rather in nested TaskSet
```

Comment:
Actually we should be able to remove this whole sentence now, since the root TaskSet is an internal one (since the tasks-under-user change).